yarn.lock

```diff
@@ -2,114 +2,184 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.5.5":
-  version "7.5.5"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.5.5.tgz#bc0782f6d69f7b7d49531219699b988f669a8f9d"
+"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.10.4":
```

Comment:
There should not be any change in yarn.lock for this PR.

Reply:
I'm updating the project version from `0.2.0` to `0.3.0` with this package addition because it wasn't updated via `lerna` previously - that produces a `yarn.lock` update by default because the dependencies haven't been checked since then.

Reply:
There do not appear to be any dependency updates between `0.2.0` and `0.3.0`, so I'm having trouble seeing how we're out of sync.

Where does this new `@babel` requirement come from?